src/colorizer/Dataset.ts

```diff
@@ -61,7 +61,7 @@ export default class Dataset {
   private features: Map<string, FeatureData>;
 
   private outlierFile?: string;
-  public outliers?: Texture | null;
+  public outliers?: Uint8Array | null;
 
```

Comment:
I think I understand this. The threejs `Texture` is opaque to the scatterplot colorizer so that's why you need the `Uint8Array`?

Reply:
Yes, exactly! The scatterplot needs access to the outlier data array. Sorry, could have explained this a bit more.